tensorflow_datasets/image/lost_and_found.py

```diff
@@ -0,0 +1,225 @@
+    download_urls = {
+        'image_left': base_url.format(self.builder_config.left_image_string)}
+    if 'image_right' in self.builder_config.features:
+      download_urls['image_right'] = base_url.format(
+          self.builder_config.right_image_string)
+    if 'segmentation_label' in self.builder_config.features \
+            or 'instance_id' in self.builder_config.features:
+      download_urls['gt'] = base_url.format('gtCoarse')
+    if 'disparity_map' in self.builder_config.features:
+      download_urls['disparity_map'] = base_url.format('disparity')
+    # split into two steps to save space for testing data
+    dl_paths = dl_manager.download(download_urls)
+    dl_paths = dl_manager.extract(dl_paths)
+
+    # point segmentation label and instance IDs both to directory for ground-truth
+    if 'gt' in dl_paths:
+      dl_paths['segmentation_label'] = dl_paths['gt']
+      dl_paths['instance_id'] = dl_paths['gt']
+
+    # first directory in the zipfile, dependent on feature to load
+    sub_dirs = {
+        'image_left': self.builder_config.left_image_string,
+        'image_right': self.builder_config.right_image_string,
+        'segmentation_label': 'gtCoarse',
+        'instance_id': 'gtCoarse',
+        'disparity_map': 'disparity'}
+
+    return [
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TRAIN,
+            # These kwargs will be passed to _generate_examples
+            gen_kwargs={feat: path.join(dl_paths[feat], sub_dirs[feat], 'train')
+                        for feat in self.builder_config.features},
+        ),
+        tfds.core.SplitGenerator(
+            name=tfds.Split.TEST,
+            # These kwargs will be passed to _generate_examples
+            gen_kwargs={feat: path.join(dl_paths[feat], sub_dirs[feat], 'test')
+                        for feat in self.builder_config.features},
+        )
+    ]
+
+  def _generate_examples(self, **paths):
+    """Yields examples."""
+    # different file-suffixes dependent on the feature to load
+    file_suffix = {
+        'image_left': self.builder_config.left_image_string,
+        'image_right': self.builder_config.right_image_string,
+        'segmentation_label': 'gtCoarse_labelIds',
+        'instance_id': 'gtCoarse_instanceIds',
+        'disparity_map': 'disparity'}
+
+    for scene_id in tf.io.gfile.listdir(paths['image_left']):
+      paths_city_root = {feat: path.join(feat_dir, scene_id)
+                         for feat, feat_dir in paths.items()}
+
+      left_city_root = paths_city_root['image_left']
+      for left_img in tf.io.gfile.listdir(left_city_root):
+        image_id = _get_id_from_left_image(left_img)
+
+        features = {
+            'image_id': image_id,
+            **{feat: path.join(paths_city_root[feat],
+                               '{}_{}.png'.format(image_id, file_suffix[feat]))
+               for feat in paths}}
+
+        yield image_id, features
+
+# Helper functions
+
```

Comment:
Please use pylint. For TF code style check this [link](https://www.tensorflow.org/community/contribute/code_style)

Reply:
I assume you would like to reduce the 2 blank lines to 1?

I checked all files with pylint using the TF code style (following the provided script in `oss_scripts/lint.sh`), however there seems to be no linter (neither pylint nor pycodestyle) that actually checks for that. Fixed it anyways.